views/js/qtiXmlRenderer/renderers/Figure.js

```diff
@@ -26,6 +26,12 @@ define(['tpl!taoQtiItem/qtiXmlRenderer/tpl/element'], function (tpl) {
                 data.tag = `${ns.name}:figure`;
             }
 
+            if (!figure.attr('showFigure')) {
+                data.tag = 'img';
+                const cleanImg = data.body.split('<qh5:figcaption >');
```

Comment:
**question:** Is it expected to have this space inside the tag? If it is an artifact of the browser, it might not be safe as it might not always be there.